generated-dockerfiles/rapidsai_centos7-devel.Dockerfile

```diff
@@ -0,0 +1,69 @@
+# RAPIDS Dockerfile for centos7 "devel" image
+#
+# RAPIDS is built from-source and installed in the base conda environment. The
+# sources and toolchains to build RAPIDS are included in this image. RAPIDS
+# jupyter notebooks are also provided, as well as jupyterlab and all the
+# dependencies required to run them.
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+ARG CUDA_VER=10.1
+ARG LINUX_VER=centos7
+ARG PYTHON_VER=3.7
+ARG RAPIDS_VER=0.17
+ARG FROM_IMAGE=rapidsai/rapidsai-dev-nightly
```

Comment:
shouldn't this be from `rapidsai/rapidsai-core-dev-nightly`?

Reply:
Ya some of the lines are still being tested and `rapidsai-core` doesn't exist yet so I need to FROM an existing image. This will be changed to what it should be once the testing is completed.